chapters/connectors.tex

```diff
@@ -616,12 +616,17 @@ \section{Restrictions of Connections and Connectors}\label{restrictions-of-conne
 
 \subsection{Balancing Restriction and Size of Connectors}\label{balancing-restriction-and-size-of-connectors}
 
-For each non-partial non-expandable connector class the number of flow variables shall be equal to the number of variables that are neither \lstinline!parameter!, \lstinline!constant!, \lstinline!input!, \lstinline!output!, \lstinline!stream! nor \lstinline!flow!.
+For each non-partial non-simple non-expandable connector class the number of flow variables shall be equal to the number of variables that are neither \lstinline!parameter!, \lstinline!constant!, \lstinline!input!, \lstinline!output!, \lstinline!stream! nor \lstinline!flow!.
 The \emph{number of variables} is the number of all elements in the connector class after expanding all records and arrays to a set of scalars of primitive types.
 The number of variables of an overdetermined type or record class (see \cref{overconstrained-equation-operators-for-connection-graphs}) is the size of the output argument of the corresponding \lstinline!equalityConstraint!() function.
+A simple connector class is not expandable has some time-varying variables, and has neither \lstinline!input!, \lstinline!output!, \lstinline!stream! nor \lstinline!flow! variables.
```

Comment:
Comma
```suggestion
A simple connector class is not expandable, has some time-varying variables, and has neither \lstinline!input!, \lstinline!output!, \lstinline!stream! nor \lstinline!flow! variables.
```

Reply:
Should it be allowed to contain more than one variable? The example from the Buildings contained only one primitive variable. My recollection from the discussion at the design meeting was that @MarkusOlssonModelon said that it would only apply to connectors containing only one component. @MarkusOlssonModelon, do you remember?

Reply:
I think I said something like that, but I don't remember why and I can't think of a reason to limit it to only one component at the moment.